app/screens/find_channels/filtered_list/filtered_list.tsx

```diff
@@ -336,7 +336,7 @@ const FilteredList = ({
     return (
         <Animated.View
             entering={FadeInDown.duration(100)}
-            exiting={FadeOutUp.duration(100)}
+            exiting={Platform.OS === 'android' ? undefined : FadeOutUp.duration(100) /* https://mattermost.atlassian.net/browse/MM-63814?focusedCommentId=178584 */}
```

Comment:
What are your thoughts in making a utility function called isAndroid(), which is available in detox test, but not here.

It would make mocking even easier in order to test this new if-else branch.

Reply:
This would apply to all the rest of the Platform.OS === 'Android' in this PR.

Reply:
In fact here is much better to use Platform.select imho

Reply:
Agree with elias about platform select.

Reply:
Or that. Looks much cleaner.